README.md

```diff
@@ -192,6 +193,13 @@ Unless a group or event identifies as political in nature, misconduct includes d
 Notwithstanding, discussions about _policies_ are generally acceptable. There is a difference between discussing policies in the context of their impact on the community and sharing reactionary comments to recent headlines. While it is acceptable to share reactions generally, reactions to political headlines or controversial topics are only accepted when a group, event, or space identifies as political in nature.
 
 
+### Disrespect of religion or spirituality
+
+Unless a group or event identifies as religious or spiritual in nature, misconduct includes proselytizing for a religious or spiritual organization or belief system in public spaces of the community. Specific spaces may be identified as preferring religious or spiritual discussion or activity, for which such proselytizing may be advertised as acceptable.
+
+It is acceptable to state or convey religious or spiritual beliefs, or advocate for creating religious or spiritual spaces in the community, permitted that such discussions and activities are in line with the rest of the code of conduct. Specifically, do not speak or act with assumption that one religious or spiritual belief is dominant, engage in political discussion outside of what is deemed acceptable in the section on politics above, repeat invitations to those who have opted out, discriminate against other beliefs, or use "religion" as an excuse to engage in misconduct named in this code of conduct.
```

Comment:
> do not speak or act with assumption that one religious or spiritual belief is dominant

Seems too broad -- might be a subtlety of what is meant by "dominant", but in most parts of the world there is a clear majority religion and spiritual practice and I don't think the intent is to prevent discussions from acknowledging that fact.

Reply:
For what it's worth, the intent here was to reference the "reinforcing social structures of domination" section.

Reply:
Any specific example(s) this is designed to cover? This is like "assuming everyone is celebrating Christmas on Dec 25" kind of behavior?